datumaro/components/converter.py

```diff
@@ -147,7 +151,12 @@ def __init__(self, extractor: IExtractor, save_dir: str, *,
         assert default_image_ext
         self._default_image_ext = default_image_ext
 
-        self._save_images = save_images
+        if save_images:
+            warnings.warn("'save-images' is deprecated and will be "
+                "removed in future. Use 'save-media' instead.",
+                DeprecationWarning, stacklevel=2)
+
+        self._save_media = save_media or save_images
```

Comment:
In which case `save_media = None`?

Reply:
Hm, I didn't notice it was set to False by default. I see a problem here that we can't check if any or both args were used.  Probably, it should just fall back to save_images by default and throw an error if both args are supplied.

Reply:
Added an error if both args are supplied

Reply:
As I said - let's make `None` the default value. Otherwise this check can't know what the user has passed to the function. The error is when both are not `None`.

Reply:
Done